docs/source/api/util.rst

```diff
@@ -22,6 +22,8 @@ Utility functions
 
 .. autofunction:: pybamm.get_parameters_filepath
 
+.. autofunction:: pybamm_install_jax
```

Comment:
This should be:
```suggestion
.. autofunction:: pybamm.install_jax
```

Reply:
What is the suggested change?
![image](https://user-images.githubusercontent.com/83979298/221203129-8247e1ed-c50b-495b-a103-2d84a63b5b6a.png)
It seems to be the same.

Reply:
`pybamm_install_jax` and `pybamm [dot] install_jax`